ThumbnailItem/ThumbnailItem.module.less

```diff
@@ -1,26 +1,91 @@
 // ThumbnailItem.module.less
 //
+@import "../styles/mixins.less";
 @import "../styles/skin.less";
 
 .thumbnailItem {
 	.thumbnail {
 		height: auto;
 		width: auto;
+	  	vertical-align: middle;
+	}
+
+	&.roundThumbnail {
+	  	.thumbnail {
+		  	border-radius: 50%;
+		}
 	}
 
 	.content {
 		overflow: hidden;
 	}
 
+	.subContent {
+		overflow: hidden;
+	}
+
+  	.slotBefore {
+	  	display: flex;
+	}
+
 	.applySkins({
 		align-items: center;
```

Comment:
If there are any css styles which do not depend on skins, please define them out of the `applySkins`. If not, those styles will be generated as many as the number of skins.